keyext.slicing/src/test/java/org/key_project/slicing/EndToEndTests.java

```diff
@@ -77,28 +79,31 @@ void sliceAgatha() throws Exception {
     @Test
     void sliceMultipleIterations() throws Exception {
         // simple Java proof
-        Pair<Proof, File> iteration1 = sliceProofFullFilename(
-            new File(testCaseDirectory,
-                "../../../../../key.ui/examples/firstTouch/05-ReverseArray/reverseArray.proof"),
+        var iteration1 = sliceProofFullFilename(
```

Comment:
This type is a bit unclear. I suggest having the full type for the first iteration and leaving the rest as `vars`

```suggestion
        Pair<Proof, Path> iteration1 = sliceProofFullFilename(
```